### PR TITLE
New Endpoint for creating new team and showing unassigned students

### LINKED ISSUE #27: Create endpoint for creating a new team and return unassigned students
### Description

Allow run owners to create a new team with at least one starting member. If members are in another existing team, move the member from that member into the new team.

Also return all unassigned students in the run config object.

### PR DESCRIPTION
**Pull request by hirokiterashima**
## Changes
- Added new endpoint for creating a new team with at least one member
- Add unassigned students in run config

## Test creating a new team
- POST to /api/teacher/run/{runId}/workgroup/create/{periodId} with an array of userId's as request data ```[1,2,3]```. It will create a new team with the members and return the id of the new team. 
- If the user is already in a team, she will be removed from the team and placed in a new team.

## Test getting unassigned students in run config
- Make a GET request to http://localhost:81/api/config/classroomMonitor/{runId}. In the response, the classmateUserInfos field should contain unassigned student id's. They appear in a block without a workgroupId.

Closes #27

### CODE REVIEW

File: src/main/java/org/wise/portal/presentation/web/controllers/InformationController.java [hunk 1]
```diff
@@ -398,15 +398,32 @@ private JSONObject getTeacherUserInfoJSONObject(Run run) {
...
+              createUserJSONInfo(user, isAllowedToViewStudentNames(run, loggedInUser));
+          studentsNotInWorkgroup.put(userJSONInfo);
+        }
+      }
+      if (studentsNotInWorkgroup.length() > 0) {
+        JSONObject studentsNotInWorkgroupInfo = new JSONObject();
+        studentsNotInWorkgroupInfo.put("periodId", period.getId());
+        studentsNotInWorkgroupInfo.put("users", studentsNotInWorkgroup);
+        classmateUserInfos.put(studentsNotInWorkgroupInfo);
+      }
```
Comment (geoffreykwan) [inline src/main/java/org/wise/portal/presentation/web/controllers/InformationController.java:425]: Should the userIds array be created and added to the object?
```studentsNotInWorkgroupInfo.put("userIds", userIds);```

**Review by breity — APPROVED** [review]
LGTM.

**Review by geoffreykwan — APPROVED** [review]
Looks good.

**Pull request merged**